build/goreleaser/linux/centos_7_arm.yml

```diff
@@ -6,7 +6,7 @@
       - linux-ctl-arm
       - linux-service-arm
     package_name: newrelic-infra
-    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el7.{{ .Arch }}"
+    file_name_template: "newrelic-infra-{{ .Env.TAG }}-1.el7.{{ .Arch }}{{ .Env.FIPS }}"
```

Comment:
Same, this should be :

`"newrelic-infra{{ .Env.FIPS }}-{{ .Env.TAG }}-1.el7.{{ .Arch }}"`

Reply:
I swapped this, should be good now